File: POO/temas/11/rb/11.rb
class Frigo
    def initialize(color="blanco")
        @color = color
        @puerta = "cerrada"
        @temperatura = 20
    end

    def abre
        @puerta = "abierta"
    end

    def cierra
        @puerta = "cerrada"
    end

    def fija_temperatura(nueva_temperatura=7)
        @temperatura = nueva_temperatura
    end

    def muestra_estado
        puts "Color: #{@color} | Puerta #{@puerta} | #{@temperatura}ºC"
    end
end


mi_frigo = Frigo.new("rojo")
mi_frigo.muestra_estado

print "Temperatura: "
t = gets.chomp.to_i

mi_frigo.fija_temperatura(t)
mi_frigo.muestra_estado

File: POO/temas/11/py/11.py
class Frigo:
    def __init__(self,color="blanco"):
        self.__color = color
        self.__puerta = "cerrada"
        self.__temperatura = 20

    def abre(self):
        self.__puerta = "abierta"

    def cierra(self):
        self.__puerta = "cerrada"

    def fija_temperatura(self,nueva_temperatura=7):
        self.__temperatura = nueva_temperatura

    def muestra_estado(self):
        print( f'Color: {self.__color} | Puerta {self.__puerta} | {self.__temperatura}ºC' )


mi_frigo = Frigo("rojo")
mi_frigo.muestra_estado()

t = int(input("Temperatura: "))

mi_frigo.fija_temperatura(t)
mi_frigo.muestra_estado()
